javascript-local-storage/data.js

```diff
@@ -0,0 +1,10 @@
+/* exported todos */
+
+var todos = [];
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+todos = JSON.parse(previousTodosJSON);
```

Comment:
don't forget to do the check for `null` before assigning the parsed value to todos